core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -222,13 +222,27 @@ public static LoadTableResponse createTable(
     throw new IllegalStateException("Cannot wrap catalog that does not produce BaseTable");
   }
 
+  /**
+   * @param catalog The catalog instance
+   * @param ident The table identifier of the table to be dropped.
+   * @deprecated Will be removed in 1.2.0, use {@link CatalogHandlers#dropTable(Catalog,
+   *     TableIdentifier, boolean)}.
+   */
+  @Deprecated
```

Comment:
Rather than adding a boolean option, can we have two methods, `dropTable` and `purgeTable`?

Reply:
done